packages/patternfly-4/react-core/src/components/AboutModal/AboutModal.test.js

```diff
@@ -48,11 +48,11 @@ test('modal does not call onClose for esc key if it is not open', () => {
   expect(props.onClose).not.toBeCalled();
 });
 
-test('Each modal is given new ariaDescribedById and ariaLablledbyId', () => {
+test('Each modal is given new aria-describedby and aria-lablledby', () => {
```

Comment:
tiny typo here: `aria-lablledby` should be `aria-labelledby`

Reply:
Good catch. Fixed in latest commit.